src/EditPokemon/EditPokemon.js

```diff
@@ -8,46 +8,27 @@ export default function EditPokemon(props) {
         updateSelectedCard,
```

Comment:
it looks like you don't need setPokemonCardData anymore now that you are using updateCardData